docs/general/administration/hardware-acceleration/intel.md

```diff
@@ -655,7 +655,27 @@ This has been tested with LXC 3.0 and may or may not work with older versions.
 
 5. Configure Jellyfin to use QSV or VA-API acceleration and change the default GPU `renderD128` if necessary.
 
-#### LXC On Proxmox
+#### LXC On Proxmox VE 8 or Newer
+
+1. Install the required drivers on the Proxmox host.
```

Comment:
A link to those drivers would be very nice here

Reply:
I agree, but I simply copied this step from the Proxmox VE < 8 instructions. For me personally, Proxmox already came with the necessary drivers (AMD), but I think intel drivers are integrated in the kernel as well. I can offer to remove this step from both instructions or elaborate further, i.e. that `/dev/dri/renderDXXX` should be available on the Proxmox host, otherwise additional drivers need to be installed.

Reply:
Please at least link to the hardware acceleration setup pages

Reply:
@felix920506 Done. I also merged the instructions for Proxmox 7 and 8, as they only differ in step 2.